opentelemetry/proto/profiles/v1development/profiles.proto

```diff
@@ -194,13 +194,13 @@ message Profile {
   // into that address range.  mapping[0] will be the main binary.
   // If multiple binaries contribute to the Profile and no main
   // binary can be identified, mapping[0] has no special meaning.
-  repeated Mapping mapping = 3;
+  repeated Mapping mapping_table = 3;
   // Locations referenced by samples via location_indices.
-  repeated Location location = 4;
+  repeated Location location_table = 4;
   // Array of locations referenced by samples.
-  repeated int64 location_indices = 15;
+  repeated int32 location_indices = 15;
```

Comment:
The only thing that gives me pause about this loss of type width is that at least some pprof profiles I know (including ones generated by internal profilers) use the address value as the location ID and 32 bits is not enough for that. But on the other hand such sparse encoding already became unsupported as we switched to the dense array indexing, so it should be fine.